application/helpers/mpdf/classes/cssmgr.php

```diff
@@ -1293,6 +1293,7 @@ function MergeCSS($inherit, $tag, $attr)
 	{
 		$p = array();
 		$zp = array();
+		if (empty($attr)) $attr = array();
```

Comment:
```suggestion
		if (empty($attr)) {
		    $attr = array();
		}
```

Sugestão: Colocar sempre chaves para melhorar legibilidade e deixar conforme padrão PSR2